### PR TITLE
Theme

### PR DESCRIPTION
**Pull request by ahamzatariq**
Applying theme functionality so user can click on theme color and the whole application changes to that color.

### CODE REVIEW

File: lib/menu_pages/items_page.dart [hunk 1]
```diff
@@ -102,16 +98,16 @@ class _ItemsPageState extends State<ItemsPage> {
...
+      // Positioned(
+      //   bottom: 16,
+      //   right: 16,
+      //   child: FloatingActionButton(
+      //     child: Icon(Icons.add),
+      //     onPressed: () {
+      //       openAddItemPopup(context);
+      //     },
+      //   ),
+      // ),
```
Comment (waleedwaseem) [inline lib/menu_pages/items_page.dart:110]: @ahamzatariq , Remove commented code.

File: lib/pages/dashboard_page.dart [hunk 2]
```diff
@@ -24,108 +26,120 @@ class _DashboardPageState extends State<DashboardPage> {
...
+                        height: 8,
+                      ),
+                      InkWell(
+                        onTap: () {
+                          setState(() {
+                            selectedItem = 3;
+                            selectedItemList[0] = false;
+                            selectedItemList[1] = false;
+                            selectedItemList[2] = false;
+                            selectedItemList[3] = true;
```
Comment (waleedwaseem) [inline lib/pages/dashboard_page.dart:123]: @ahamzatariq , Please fix this in the initial PR and backmerge the code all the way here.

Comment (ahamzatariq) [reply lib/pages/dashboard_page.dart:123]: Fixed. But didn't backmerge.

File: lib/menu_pages/items_page.dart (hunk 1 above)
Comment (ahamzatariq) [reply lib/menu_pages/items_page.dart:110]: It's already removed in the later commits.

**Pull request merged**